build-aux/toolchain/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Build the toolchain
+
+# Stop this script if error occur
+set -e
+
+# Check dependency is install
+source ./build-aux/toolchain/build/dependencies.sh
+
+# Load the configuration file
+source ./build-aux/toolchain/config.sh
+
+# Create the toolchain install directory
+if [ ! -d  $INSTALL_DIRECTORY ];  then 
```

Comment:
`mkdir -pv $INSTALL_DIRECTORY` is better here IMO, since it won't fail if the directory already exists and will log its creation on stdout.

Reply:
done